### PR TITLE
UTOPIA-217: Init ReactJS Project

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Inits a new ReactJS project that uses TypeScript and Vite

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Init'ed the project and ran `npm run dev` to test that it worked.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by akroon3r — CHANGES_REQUESTED** [review]
let's clean up the directory structure here by getting rid of the /dpia-app directory to have all the source code in the /src/frontend directory instead please. 🚀

File: src/frontend/dpia-app/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.17",
+    "@types/react-dom": "^18.0.6",
+    "@vitejs/plugin-react": "^2.1.0",
+    "typescript": "^4.6.4",
+    "vite": "^3.1.0"
+  }
+}
```
Comment (akroon3r) [inline src/frontend/dpia-app/package.json:22]:
```suggestion
}

```

**Comment by Bottle7** [comment]
@akroon3r Changes have been made!

**Review by akroon3r — APPROVED** [review]
Looking good - great first step team

**Comment by binggaofw** [comment]
looks for me, just one question, do we add Jest in this PR or later

**Comment by Bottle7** [comment]
> looks for me, just one question, do we add Jest in this PR or later @Bottle7

Considering the granularity of the tickets--like how the Dockerfile creation is a separate ticket--I would say that the testing stuff will be added to the project separately.

**Review by akroon3r — APPROVED** [review]

**Comment by akroon3r** [comment]
@binggaofw on you to approve 👍

**Review by antsand — APPROVED** [review]
Looks Good!

**Pull request merged**